roles/ceph-mon/tasks/docker/start_docker_monitor.yml

```diff
@@ -10,6 +10,48 @@
 - name: pull ceph daemon image
   shell: "docker pull {{ ceph_mon_docker_username }}/{{ ceph_mon_docker_imagename }}"
 
+# Use systemd to manage container on Atomic host
+- name: generate systemd unit file
+  sudo: true
+  config_template:
+    src: ceph-mon.service.j2
+    dest: /var/lib/ceph/ceph-mon@.service
+    owner: "root"
+    group: "root"
+    mode: "0644"
+    config_overrides: {}
+    config_type: ini
+
+- name: link systemd unit file for mon instance
+  file:
+    src: /var/lib/ceph/ceph-mon@.service
+    dest: /etc/systemd/system/multi-user.target.wants/ceph-mon@{{ ansible_hostname }}.service
+    state: link
+  when: is_atomic
+
+- name: enable systemd unit file for mon instance
+  shell: systemctl enable /etc/systemd/system/multi-user.target.wants/ceph-mon@{{ ansible_hostname }}.service
```

Comment:
`service` doesn't seem to work. It looks for a file under `/usr`, but since we cannot write to `/usr` on Atomic (it is readonly), we have to do our own poor man's `service`.